config/locales/exercises/en.yml

```diff
@@ -6,4 +6,6 @@ en:
       hints_reference: "If you get stuck on the exercise, check out `HINTS.md`, but try and solve it without using those first :)"
       tests_intro: "Running the tests"
       submitting_solution_intro: "Submitting your solution"
+      submitting_solution: "You can submit your solution using the `exercism submit %{solution_file_paths}` command.\nThis command will upload your solution to the Exercism website and print the solution page's URL.\n\nIt's possible to submit an incomplete solution which allows you to:\n\n- See how others have completed the exercise\n- Request help from a mentor"
       help_intro: "Need to get help?"
+      help: "If you'd like help solving the exercise, check the following pages:\n\n- The [%{track_title} track's documentation](https://exercism.io/docs/tracks/%{track_slug})\n- [Exercism's support channel on gitter](https://gitter.im/exercism/support)\n- The [Frequently Asked Questions](https://exercism.io/docs/faq) TODO: (Required) use correct link\n\nShould those resources not suffice, you could submit your (incomplete) solution to request mentoring."
```

Comment:
I've linked to three resources that students can use for help:

- The track docs (e.g. https://exercism.lol/docs/tracks/csharp). That page probably needs a bit of love, but it offers a great starting point to find track-specific resources
- The gitter support channel
- The FAQs

Reply:
As a final option (and purposefully the last one listed), I've suggested that people can submit incomplete solutions to request mentoring.

Reply:
OK, great :)

Reply:
You can't submit incomplete solutions via the editor. So just confirming that  `help` is only shown to CLI users?

Reply:
That is correct